2025/incoming/all-i-want-for-christmas-is-the-right-aspect-ratio.pod

```diff
@@ -0,0 +1,459 @@
+Author: Dave Cross <dave@perlhacks.com>
+Title: All I Want for Christmas Is the Right Aspect Ratio
+Topic: App::BlurFill
+
+=encoding utf8
+
+=head2 A bad start to Christmas
+
+=for :html
+<img src="pixie-the-elf_blur.png">
+
+Pixie the Elf (whose parents had strange ideas about names) was not having
+a good December.
+
+This year she’d been seconded from Santa’s Workshop to the North Pole
+Communications Department. Her job was to prepare a huge batch of festive
+images: hero banners for the NaughtyOrNice dashboard, thumbnails for the
+Sleigh Tracking app, preview cards for Elfbook, and even a few social posts
+for Santa’s surprisingly popular “Reels from the Sleigh”.
+
+All of those systems wanted I<different aspect ratios>.
+
+The NaughtyOrNice dashboard wanted 16:9. The old in-house CMS insisted on 4:3.
+Elfbook square-cropped everything. One particularly opinionated goblin had
+even insisted on a tall “story” format.
+
+And all Pixie had were a couple of folders of lovely but badly sized photos:
+Santa in front of the Aurora, reindeer in flight, close-ups of toys, and the
```

Comment:
Aurora is the name of Grok's image generator, so people might get the wrong idea about what's about to happen.